BENCHMARKING.md

```diff
@@ -107,7 +113,80 @@ for i in $(seq 0 $(ssget -n)); do
 done
 ```
 
-### 3: Benchmarking overview
+### 3: Running benchmarks manually
+When compiling `ginkgo` with the flag `-DGINKGO_BUILD_BENCHMARKS=on`, a suite of
+executables will be generated depending on the `CMake` configuration. These
```

Comment:
I would write either `cmake` (all lowercase and refer to the command line) or CMake (no code marking since it is a program).
```suggestion
executables will be generated depending on the CMake configuration. These
```